qiskit/providers/__init__.py

```diff
@@ -110,6 +109,394 @@
    Job
    JobV1
 
+======================
+Writing a New Provider
+======================
+
+If you have a quantum device or simulator that you would like to integrate with
+Qiskit you will need to write a provider. A provider will provide Terra with a
+method to get available :class:`~qiskit.provider.BackendV1` objects. The
+:class:`~qiskit.provider.BackendV1` object provides both information describing
```

Comment:
It might be a good idea to make this into a list instead. It could help writers of providers understand what their set of responsibilities are

Reply:
Hmm, I just read to the list below which I think is super clear. So maybe keep this the same format. I suggested below to break up the sentences